src/Options.php

```diff
@@ -270,19 +270,29 @@ public function setInAppIncludedPaths(array $paths): void
 
     /**
      * Gets the logger used by Sentry.
+     *
+     * @deprecated since version 3.2, to be removed in 4.0
      */
-    public function getLogger(): string
+    public function getLogger(/*bool $triggerDeprecation = true*/): string
     {
+        if (0 === \func_num_args() || (\func_num_args() > 0 && false !== func_get_arg(0))) {
```

Comment:
Double-checking the number of arguments is superflouous
```suggestion
        if (0 === \func_num_args() || false !== func_get_arg(0))) {
```